From 01dcdab682125baf924436da917f45992e16e829 Mon Sep 17 00:00:00 2001
From: Yuxi Sun <b36102@freescale.com>
Date: Wed, 4 May 2011 18:08:33 +0800
Subject: [PATCH 221/442] ENGR00142978 unit_test: mxc_v4l2_capture.out
 Segmentation fault

fix Segmentation fault issue when running mxc_v4l2_capture.out test
with out given a record file name

Signed-off-by: Yuxi Sun <b36102@freescale.com>
---
 test/mxc_v4l2_test/mxc_v4l2_capture.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/mxc_v4l2_test/mxc_v4l2_capture.c b/test/mxc_v4l2_test/mxc_v4l2_capture.c
index 9a8dc2a..3b0906a 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_capture.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_capture.c
@@ -274,7 +274,7 @@ int v4l_capture_test(int fd_v4l, const char * file)
         FILE * fd_y_file = 0;
         int count = g_capture_count;
 
-        if ((fd_y_file = fopen(file, "wb")) < 0)
+        if ((fd_y_file = fopen(file, "wb")) ==NULL)
         {
                 printf("Unable to create y frame recording file\n");
                 return -1;
-- 
1.8.0

